test/functionalTest/cases/4149_pagination_for_request_forwarding/pagination_for_request_forwarding.test

```diff
@@ -0,0 +1,2532 @@
+echo
+echo
+
+
+
+echo "19. Create entity C4/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C4",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+
+echo "20. Create entity C5/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C5",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+
+echo "21. Create entity C6/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C6",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+
+echo "22. Create entity C7/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C7",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+
+echo "23. Create entity C8/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C8",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+echo "24. Create entity C9/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C9",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+
+echo "25. Create entity C10/T with attribute A2, in CP2"
+echo "================================================"
+payload='{
+  "id": "C10",
+  "type": "T",
+  "A2": { "value": 2 }
+}'
+orionCurl --url /v2/entities --payload "$payload" --port $CP2_PORT
+echo
+echo
+
+
+
+echo "26. Register entities of type T, with ID .*, and attribute A1, for CP1"
+echo "======================================================================"
+payload='{
+  "dataProvided": {
+    "entities": [
+      {
+        "type": "T",
+        "idPattern": ".*"
+      }
+    ],
+    "attrs": [ "A1" ]
+  },
+  "provider": {
+    "http": {
+      "url": "http://localhost:'${CP1_PORT}'/v2"
+    }
+  }
+}'
+orionCurl --url /v2/registrations --payload "$payload"
+echo
+echo
+
+
+
+echo "27. Register entities of type T, with ID .*, and attribute A2, for CP2"
+echo "======================================================================"
+payload='{
+  "dataProvided": {
+    "entities": [
+      {
+        "type": "T",
+        "idPattern": ".*"
+      }
+    ],
+    "attrs": [ "A2" ]
+  },
+  "provider": {
+    "http": {
+      "url": "http://localhost:'${CP2_PORT}'/v2"
+    }
+  }
+}'
+orionCurl --url /v2/registrations --payload "$payload"
+echo
+echo
+
+
+
+echo "28. GET /v2/entities with limit<5 and check no limit forwarded to CP"
```

Comment:
To be precise:

```suggestion
echo "28. GET /v2/entities with limit=3 and check no limit forwarded to CP"
```

Reply:
Fixed in [1e47b07](https://github.com/telefonicaid/fiware-orion/pull/4275/commits/1e47b077ce86f15d09438331b764710751bf1910)